docs/wallet_browser_extension/intro.md

```diff
@@ -0,0 +1,53 @@
+# ParallelChain Wallet Extension
+
+ParallelChain Wallet Extension is an extension for managing your XPLL and integrating your DApp.
+It allows your DApp to interact with your DApp users' XPLL accounts and make smart contract calls through the extension.
+
+# Features
+
+- Browser Extension
+  - Managing XPLL Wallet (Like in [Explorer Wallet](https://explorer.parallelchain.io/))
+    - Sending Transactions
+    - Staking
+  - Triggering Confirmation for smart contract or send token calls
+- Inject Global Javascript API into browser for DApps to interact with user wallets
+  - Properties
```

Comment:
What does properties and events here mean?

Reply:
The introduction seems out-of-date, move the `properties` and `event` detail to its own section. 
Solved by 0486a15.